src/controllers/scripting/controllerscriptenginebase.cpp

```diff
@@ -36,6 +36,11 @@ void ControllerScriptEngineBase::registerTrackCollectionManager(
     s_pTrackCollectionManager = std::move(pTrackCollectionManager);
 }
 
+void ControllerScriptEngineBase::registerPlayerManager(
```

Comment:
This function is never called. Will this be expected to be whenever a controller starts? From ControllerManager?

In this case, how about changing the singleton pattern to simple dependency injection, when you [start the engine](https://github.com/mixxxdj/mixxx/blob/0d9fe85e3b923c69cba6a968ce82a2f35b2bd25c/src/controllers/controller.cpp#L58)?